docs/sdkx_android/migration-guide.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 sidebar_position: 140
 title: Migration Guide
-description: "SDK X is a Hybrid SDK that helps you roll out innovation faster by enabling most of the updates over the air that flow to end users without any downtime or app updates."
```

Comment:
I think this string was provided by product in migration guide.
We should check once before removing completely.

Reply:
This string is still present in the Intro section of this page. I thought it made the description too wordy, so removed it.